Library/ViewModels/DiscoveryPostcardViewModel.swift

```diff
@@ -346,6 +403,11 @@ public final class DiscoveryPostcardViewModel: DiscoveryPostcardViewModelType,
     self.userSessionEndedProperty.value = ()
   }
 
+  fileprivate let enableProjectCategoryExperiment = MutableProperty<Bool>(false)
```

Comment:
Another convention we use is to append the word `Property` to the MutableProperty name. This way, is easier to identify what's MutableProperty and what's signal on the code. So `enableProjectCategoryExperimentProperty` would be better.